### PR TITLE
55 rework sdk to utilize single job submission queue instead of one per workflow type

### PR DESCRIPTION
**Pull request by lfse-slafleur**
Together with https://github.com/Project-OMOTES/orchestrator/pull/60

### CODE REVIEW

**Review by MarkTNO — APPROVED** [review]

File: src/omotes_sdk/queue_names.py [hunk 1]
```diff
@@ -14,22 +15,21 @@ def omotes_exchange_name() -> str:
...
-        :param workflow_type: Workflow type.
         :return: The queue name.
         """
-        return f"job_submissions.{workflow_type.workflow_type_name}"
+        return "job_submissions"
 
     @staticmethod
-    def job_results_queue_name(job: Job) -> str:
+    def job_results_queue_name(job_uuid: uuid.UUID) -> str:
         """Generate the job results queue name given the job.
```
Comment (cwang39403) [inline src/omotes_sdk/queue_names.py:27]: Nitpicking: "Generate the job results queue name given the job **id**"

File: src/omotes_sdk/queue_names.py [hunk 2]
```diff
@@ -14,22 +15,21 @@ def omotes_exchange_name() -> str:
...
 
-        :param job: The job.
+        :param job_uuid: The identifier of the job.
         :return: The queue name.
         """
-        return f"jobs.{job.id}.result"
+        return f"jobs.{job_uuid}.result"
 
     @staticmethod
     def job_progress_queue_name(job: Job) -> str:
```
Comment (cwang39403) [inline src/omotes_sdk/queue_names.py:35]: Is there a reason to have `job_progress_queue_name()` and `job_status_queue_name()` both accept `Job` to generate the queue name, while `job_results_queue_name()` is now accepting `job_uuid`

**Review by cwang39403 — APPROVED** [review]
Approve with minor comments

File: src/omotes_sdk/queue_names.py (hunk 2 above)
Comment (lfse-slafleur) [reply src/omotes_sdk/queue_names.py]: Good idea to make it consistent: https://github.com/Project-OMOTES/omotes-sdk-python/pull/61/commits/c239f062aedb1ad419a44d4cd35fb65e2fa30806

File: src/omotes_sdk/queue_names.py (hunk 1 above)
Comment (lfse-slafleur) [reply src/omotes_sdk/queue_names.py]: Fixed in https://github.com/Project-OMOTES/omotes-sdk-python/pull/61/commits/c239f062aedb1ad419a44d4cd35fb65e2fa30806

**Pull request merged**